cairo/ethereum/cancun/fork.cairo

```diff
@@ -732,3 +754,66 @@ func _check_versioned_hashes_version{range_check_ptr}(
 
     return ();
 }
+
+func get_last_256_block_hashes{
+    range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: KeccakBuiltin*
+}(chain: BlockChain) -> ListHash32 {
+    alloc_locals;
+
+    // If no blocks, return empty array
+    if (chain.value.blocks.value.len == 0) {
+        let (empty_hashes_alloc: Hash32*) = alloc();
+        tempvar empty_hashes = ListHash32(new ListHash32Struct(data=empty_hashes_alloc, len=0));
+        return empty_hashes;
+    }
+
+    // Get last 255 blocks or all blocks if less than 255
+    let less_than_255 = is_le_felt(chain.value.blocks.value.len, 255);
```

Comment:
```suggestion
    let is_le_255 = is_le(chain.value.blocks.value.len, 255);
```

The input should not be filled with > 2**128 block here

(nit: name: `less_than` but using `is_le`